server/server.js

```diff
@@ -7,34 +7,42 @@ const passport = require('passport');
 const cookieParser = require('cookie-parser');
 const session = require('express-session');
 const dotenv = require('dotenv');
-
-const users = require('./routes/api/user');
+// const FileStore = require('session-file-store')(session);
```

Comment:
This should be removed

Reply:
> I will actually prefer will go for session-file, I changed to Mongo Store because of the bugs we were having.
> @enyeInc / @Way2nnadi what do you advise we go for, a file store or a DB store for sessions relating this to your advice on reducing DB request?

Reply:
I don't feel a **file** store will be as efficient as a mongodb store though 🤔. Using files to store isn't really a great idea. Plus since you set **resave** to false in the session configuration, there will be more reads and lesser writes.

Reply:
You have a great point. See fix below